src/components/feature-hints/feature-hints.js

```diff
@@ -15,7 +16,10 @@ import './feature-hints.scss';
 
 const localStorageKeyShowHints = 'showFeatureHints';
 export const localStorageKeyFeatureHintsStep = 'featureHintStep';
-const numFeatureHints = featureHintsContent.length;
+const updatedFeatureHintsContent = isRunningLocally()
```

Comment:
This is not a blocker but just for clarity--  from what I understand 'Publish and share Kedro-viz' feature hint shouldn't show in the deployed version and hence we have to write this code. 

But this feature hint only gets shown when the user hovers over the 'deploy button' and there won't be any deploy button, so why do we still need to have this check here?

Reply:
Because even if the indicator dot isn't there the content in the card would still exist.